sklearn/pipeline.py

```diff
@@ -1623,8 +1644,9 @@ def transform(self, X):
             The `hstack` of results of transformers. `sum_n_components` is the
             sum of `n_components` (output dimension) over transformers.
         """
+        params = Bunch(transform={})
```

Comment:
Maybe a small temporary `# TODO` mentioning that `transform` should accept in the future metadata via a `**params` in the signature?